src/coprocessor/dag/expr/builtin_string.rs

```diff
@@ -61,55 +61,49 @@ impl ScalarFunc {
         Ok(Some(input.len() as i64))
     }
 
+    #[inline]
+    fn locate(s: &str, p: &str, pos: usize) -> i64 {
+        s.find(p)
```

Comment:
How about using `twoway::find_str` because it provides SSE4.2 acceleration compared to std's `find`?

Reply:
OK. Nextime I will check the documentation of crates more carefully 😄 Thx!